mailet/base/src/test/java/org/apache/mailet/base/test/FakeMail.java

```diff
@@ -148,11 +151,20 @@ public Builder recipients(MailAddress... recipients) {
             return this;
         }
 
+        public Builder recipients(String... recipients) {
+            Arrays.stream(recipients).forEach(Throwing.consumer(this::recipient));
+            return this;
+        }
+
         public Builder recipient(MailAddress recipient) {
             this.recipients.add(recipient);
             return this;
         }
 
+        public Builder recipient(String recipient) throws AddressException {
```

Comment:
I think this one is useless, it's going to be handle by ``public Builder recipients(String... recipients)`` anyway so I will make ``public Builder recipients(String... recipients)`` directly use ``public Builder recipient(MailAddress recipient)``

Reply:
No because `recipient != recipients`. The plural makes a difference. Feat @mbaechler  who asked me months ago for this.

Reply:
I factorized calls

Reply:
`recipient != recipients` is it a joke ?

Reply:
No. Plural VS singular.

The intent is to make tests more readable.

And that was a PR comment ;-)